src/wp-admin/includes/image-edit.php

```diff
@@ -391,6 +397,11 @@ function wp_save_image_file( $filename, $image, $mime_type, $post_id ) {
 				return imagepng( $image, $filename );
 			case 'image/gif':
 				return imagegif( $image, $filename );
+			case 'image/webp':
+				if ( function_exists( 'imagewebp' ) ) {
+					return imagewebp( $image, null, apply_filters( 'jpeg_quality', 90, 'edit_image' ) );
```

Comment:
```suggestion
					/** This filter is documented in wp-includes/class-wp-image-editor.php */
					return imagewebp( $image, null, apply_filters( 'jpeg_quality', 90, 'edit_image' ) );
```

Reply:
Ah, thanks. Also, I think I should lower this setting, it looks too high.